Content.Client/Radiation/Components/GeigerVisualsComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Shared.Radiation.Components;
+
+namespace Content.Client.Radiation.Components;
+
+/// <summary>
+///     Visualizer for generic geiger counter.
+/// </summary>
+[RegisterComponent]
+public sealed class GeigerVisualsComponent : Component
+{
+    /// <summary>
+    ///     Sprite states mapped by radiation danger level.
+    /// </summary>
+    [DataField("states")]
+    public Dictionary<GeigerDangerLevel, string> States = new();
+}
```

Comment:
Is this intended to be used outside of geigercomponent?

Reply:
Not all geiger counters have visualization logic (ie. clothing). If appearance system will be subscribed to geiger component, it will be called when other system has changed it appearance.

It won't do anything, because no system has set `GeigerVisuals`. So I use `GeigerVisualsComponent` to mark what geiger counters need to be updated by appearance system.